app/views/shared/_user_approvals.html.erb

```diff
@@ -1,55 +1,57 @@
-<% if @users_requesting_approval.any? && policy([:manage, :user, @users_requesting_approval.first]).enable_access? %>
-  <% if @users_requesting_approval.any? %>
+<div class="card d-flex mb-2">
```

Comment:
Moved `<div class="card...">` into the partial. User approvals should always be rendered within a card.